pylint/plugins/hass_enforce_greek_micro_char.py

```diff
@@ -0,0 +1,48 @@
+"""Plugin for checking correct micro unicode card is used."""
+
+from __future__ import annotations
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.lint import PyLinter
+
+
+class HassEnforceGreekMicroCharChecker(BaseChecker):
+    """Checker for micro char."""
+
+    name = "hass-enforce-greek-micro-char"
+    priority = -1
+    msgs = {
+        "W7452": (
+            "Constants with the mico sign must be encoded as U+03BC (\u03bc), not as U+00B5 (\u00b5)",
+            "hass-enforce-greek-micro-char",
+            "According to [The Unicode Consortium](https://en.wikipedia.org/wiki/Unicode_Consortium), the Greek letter character is preferred [10].",
```

Comment:
This link points to some general information about the unicode consortium, can we instead link to a source saying  U+03BC is preferred over U+00B5 for the micro unit prefix?
Maybe better to link to https://en.wikipedia.org/wiki/Micro-, the second paragraph says:
> In [Unicode](https://en.wikipedia.org/wiki/Unicode), the symbol is represented by U+03BC μ GREEK SMALL LETTER MU or the legacy symbol U+00B5 µ MICRO SIGN.

Reply:
Updated the link